cmd/aws-s3-provisioner.go

```diff
@@ -169,15 +169,15 @@ func (p awsS3Provisioner) createBucket(name string) (*v1alpha1.Connection, error
 	glog.Infof("Bucket %s successfully created", name)
 	return  &v1alpha1.Connection{
 		&v1alpha1.Endpoint{
-			BucketHost: s3Host,
+			BucketHost: fmt.Sprintf("%s//:%s", s3Host, s3Domain),
```

Comment:
Is there a reason to break the host into `s3` and `.amazon.aws`?  The host portion of the url is made up of the domain name `s3.amazonaws` and the top level domain `.com`.

Reply:
no reason really. The code just started out that way...